platform/ios/.gitignore

```diff
@@ -47,3 +47,54 @@ mapbox-gl-native.code-workspace
 /scripts/generate-shaders.list
 /scripts/generate-style-code.list
 *.patch
+
+
+# node platform/darwin/scripts/generate-style-code.js
+# generated file list will be in:
+# platform/ios/platform/darwin/scripts/generate-style-code.list
+
+platform/darwin/src/MGLLight.h
```

Comment:
This list would need to be updated any time the codegen scripts change, such as any time there’s a new layer type. It would be more maintainable to move these generated files into a generated/ subdirectory.